internal/test_helpers/fixtures/produce/pass

```diff
@@ -0,0 +1,932 @@
+[33m[tester::#UM3] [Decoder] [0m[36m    - .min_version (0)[0m
+[33m[tester::#UM3] [Decoder] [0m[36m    - .max_version (0)[0m
+[33m[tester::#UM3] [Decoder] [0m[36m    - .TAG_BUFFER[0m
+[33m[tester::#UM3] [Decoder] [0m[36m  - .ApiKeys[58][0m
+[33m[tester::#UM3] [Decoder] [0m[36m    - .api_key (75)[0m
+[33m[tester::#UM3] [Decoder] [0m[36m    - .min_version (0)[0m
+[33m[tester::#UM3] [Decoder] [0m[36m    - .max_version (0)[0m
+[33m[tester::#UM3] [Decoder] [0m[36m    - .TAG_BUFFER[0m
+[33m[tester::#UM3] [Decoder] [0m[36m  - .ApiKeys[59][0m
+[33m[tester::#UM3] [Decoder] [0m[36m    - .api_key (80)[0m
+[33m[tester::#UM3] [Decoder] [0m[36m    - .min_version (0)[0m
+[33m[tester::#UM3] [Decoder] [0m[36m    - .max_version (0)[0m
+[33m[tester::#UM3] [Decoder] [0m[36m    - .TAG_BUFFER[0m
+[33m[tester::#UM3] [Decoder] [0m[36m  - .ApiKeys[60][0m
+[33m[tester::#UM3] [Decoder] [0m[36m    - .api_key (81)[0m
+[33m[tester::#UM3] [Decoder] [0m[36m    - .min_version (0)[0m
+[33m[tester::#UM3] [Decoder] [0m[36m    - .max_version (0)[0m
+[33m[tester::#UM3] [Decoder] [0m[36m    - .TAG_BUFFER[0m
+[33m[tester::#UM3] [Decoder] [0m[36m  - .throttle_time_ms (0)[0m
+[33m[tester::#UM3] [Decoder] [0m[36m  - .TAG_BUFFER[0m
+[33m[tester::#UM3] [0m[92m✓ correlation_id: 177586623[0m
+[33m[tester::#UM3] [0m[92m✓ ErrorCode: 0 (NO_ERROR)[0m
+[33m[tester::#UM3] [0m[92m✓ API keys array length: 61[0m
+[33m[tester::#UM3] [0m[92m✓ MinVersion for PRODUCE is <= 11 & >= 0[0m
+[33m[tester::#UM3] [0m[92m✓ MaxVersion for PRODUCE is >= 11[0m
+[33m[tester::#UM3] [0m[92m✓ MinVersion for API_VERSIONS is <= 4 & >= 0[0m
+[33m[tester::#UM3] [0m[92m✓ MaxVersion for API_VERSIONS is >= 4[0m
+[33m[tester::#UM3] [0m[92mTest passed.[0m
+[33m[tester::#UM3] [0m[36mTerminating program[0m
+[33m[tester::#UM3] [0m[36mProgram terminated successfully[0m
+
+[33m[tester::#CK2] [0m[94mRunning tests for Stage #CK2 (ck2)[0m
+[33m[tester::#CK2] [Serializer] [0m[36mWriting log files to: /tmp/kraft-combined-logs[0m
+[33m[tester::#CK2] [Serializer] [0m[36m  - Wrote file to: /tmp/server.properties[0m
+[33m[tester::#CK2] [Serializer] [0m[36m  - Wrote file to: /tmp/kraft-combined-logs/meta.properties[0m
+[33m[tester::#CK2] [Serializer] [0m[36m  - Wrote file to: /tmp/kraft-combined-logs/.kafka_cleanshutdown[0m
+[33m[tester::#CK2] [Serializer] [0m[36m  - Wrote file to: /tmp/kraft-combined-logs/bar-0/partition.metadata[0m
+[33m[tester::#CK2] [Serializer] [0m[36m  - Wrote file to: /tmp/kraft-combined-logs/foo-0/partition.metadata[0m
+[33m[tester::#CK2] [Serializer] [0m[36m  - Wrote file to: /tmp/kraft-combined-logs/pax-0/partition.metadata[0m
+[33m[tester::#CK2] [Serializer] [0m[36m  - Wrote file to: /tmp/kraft-combined-logs/pax-1/partition.metadata[0m
+[33m[tester::#CK2] [Serializer] [0m[36m  - Wrote file to: /tmp/kraft-combined-logs/paz-0/partition.metadata[0m
```

Comment:
@ryan-gang we need to trim these logs down, the volume here is crazy - looks like this is only because we're creating topics/partitions for ALL stages in the same place?